provider/resources.go

```diff
@@ -18,14 +18,21 @@ import (
 	"fmt"
 	"path/filepath"
 	"unicode"
+    _ "embed"
```

Comment:
The indentation looks fine locally, not sure if this is because of Github's tabs vs spaces display.